e2e/config/config.go

```diff
@@ -102,7 +107,8 @@ type RPCs struct {
 	EVM               string     `yaml:"evm"`
 	Bitcoin           BitcoinRPC `yaml:"bitcoin"`
 	Solana            string     `yaml:"solana"`
-	TONSidecarURL     string     `yaml:"ton_sidecar_url"`
+	TON               string     `yaml:"ton_liteserver_config"`
```

Comment:
```suggestion
	TON               string     `yaml:"ton_liteserver_config"`
```
Shouldn't we keep the yaml field consistent with field name?